packages/vue-renderer/src/renderers/ssr.js

```diff
@@ -140,12 +151,17 @@ export default class SSRRenderer extends BaseRenderer {
     if (shouldInjectScripts) {
       APP += this.renderScripts(renderContext)
     }
+
+    // Append body scripts
+    APP += m.meta.text({ body: true })
+    APP += m.link.text({ body: true })
+    APP += m.style.text({ body: true })
     APP += m.script.text({ body: true })
     APP += m.noscript.text({ body: true })
 
     // Template params
     const templateParams = {
-      HTML_ATTRS: 'data-n-head-ssr ' + m.htmlAttrs.text(),
+      HTML_ATTRS: m.htmlAttrs.text(true),
```

Comment:
Can we annotate param like `true /* prepend */`

Reply:
I'd favor a variable like `shouldPrepend`

Reply:
The `true` here doesnt depict prepend / append but whether to automatically add the [`ssrAttribute`](https://vue-meta.nuxtjs.org/api/#ssrattribute) or not